test/safe/push.test.ts

```diff
@@ -548,7 +605,22 @@ describe("Push Webhook", () => {
 				expect.anything()
 			).mockResolvedValue(true);
 
-			await expect(app.receive(event)).toResolve();
+			mockGithubAccessToken();
+
+			const event = createPushEventAndMockRestReqeustsForItsProcessing();
+
+			console.log("Date now: " + Date.now())
+			await app.receive(event);
+
+			await waitUntil( async () => {
+				// eslint-disable-next-line jest/no-standalone-expect
+				expect(githubNock.pendingMocks()).toEqual([]);
+				// eslint-disable-next-line jest/no-standalone-expect
+				expect(jiraNock.pendingMocks()).toEqual([]);
```

Comment:
Do we still need this expect statements? There is another PR from @mboudreau , that has a afterEach block to check all nocks have been completed.

Reply:
As discussed F2F, we still need these checks within the test, because here we actively wait until we call Jira. 

When we receive a webhook we don't call Jira, but send message on the queue and respond with 200. 

If we won't wait here until Jira is called and exist the test, the global check for nock.isDone will fail, because we didnt' call Jira yet.

So that's why we have to actively wait until the message on the queue will be executed and then exit the test. 
This function has a timeout and fail if it won't make it fast enough.